src/model/Computation.js

```diff
@@ -62,6 +62,9 @@ export default class Computation extends Model {
 
 		this.boundsSensitive = true;
 		this.dirty = true;
+
+		// computations don't shuffle
+		this.shuffle = undefined;
```

Comment:
After having slept on it, nothing really jumped out as a best solution, so I punted with a TODO for now.